cmd/crio/main.go

```diff
@@ -372,7 +372,7 @@ func main() {
 		},
 		cli.StringSliceFlag{
 			Name:  "registry",
-			Usage: fmt.Sprintf("registry to be prepended when pulling unqualified images, can be specified multiple times (default: %q)", defConf.Registries),
+			Usage: fmt.Sprintf("registry to be prepended when pulling unqualified images, can be specified multiple times (default: configured in /etc/containers/registries.conf)"),
```

Comment:
Do we want a similar comment about the default for `--insecure-registry`?

Reply:
Not exactly the same comment, because `--registry` directly overrides the list in `registries.conf`, but `--insecure-registry` adds new CRI-O-implemented items without disabling/overriding the insecure registries/repos configured in `registries.conf`.

Having a `--help` entry that says “this option does $FOO (but another config file also allows $FOO)” seems unexpected to me; it would be reasonable to put in a man page or other primary reference documentation, but it seems a bit much for the hopefully-concise `--help` output. (The `--insecure-registry` entry is already, at least, borderline, in that aspect.)

Reply:
ah makes sense. let's leave it as is then